command/ca/provisionerbeta/provisioner.go

```diff
@@ -170,6 +170,35 @@ var (
 		Usage: `Always set the common name in provisioned certificates.`,
 	}
 
+	// SCEP provisioner flags
```

Comment:
When defining flags that have a value try to us the type in the description inside `<>`, like so: https://github.com/smallstep/cli/blob/6b23538520fc8c720b067bec2fb7fe630f8825f4/flags/flags.go#L251-L257. That's become sort of a standard.

Reply:
According to the usage at the top, the type for the challenge flag is `<challenge>`.

Reply:
👍 

Will try to keep this in mind for future additions.